tasks/stache.js

```diff
@@ -1254,7 +1342,7 @@ module.exports = function (grunt) {
 
                 // Module wasn't found, so let's install this module in Stache's root.
                 case false:
-                    utils.log("Module " + module + " not found. Attempting to locate in Stache's root...");
+                    slog.warning("Module " + module + " not found. Attempting to locate in Stache's root...");
```

Comment:
I believe this may litter NPM2 installations with warnings, that are also not clear if it was successful.  Thoughts on if this condition exists, check the Stache root directory for the task, then displaying the warning (or even as an error at this point)?